.travis.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     # Run unit tests.
   - script: make test-unit
     # Run e2e tests.
-  - script: ./scripts/travis-e2e.sh
```

Comment:
Do we not want to remove the old file?

Reply:
No, the consensus was that we want to save everything related to `minikube` until `kind` starts automatically publishing images.